src/components/sidebar/Sidebar.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import Instructions from './Instructions';
+
+const Sidebar: React.FC = function(){
+    return(
+        <div>
+            <Instructions/>
+            <button className='start-now-button' type='button'>Start now</button>
```

Comment:
What happens when this button is clicked? When does that functionality come into play?

Reply:
The button functionality will be implemented on the next ticket.